geoips/compare_outputs.py

```diff
@@ -197,7 +197,7 @@ def get_out_diff_fname(compare_product, output_product, ext=None, flag=None):
 
 
 def images_match(output_product, compare_product, fuzz="5%"):
-    """Use imagemagick compare system command to compare two images.
+    """Use PIL and matplotlib to compare two images.
```

Comment:
This line should actually be 'Use PIL and Numpy to compare to images'.

Reply:
Also, we can remove the 'fuzz' argument as it is no longer used.

Reply:
I updated the docstring, and made a note that fuzz is currently not implemented.  I think we will want to re-implement that at a later date, to allow slight variations in the imagery still passing (that is the end goal for having the numpy-based comparisons, so we have more control over what constitutes a "match").

@evrose54 Please resolve this conversation if that looks ok to you.

Reply:
That sounds great to me. You're right that it's probably a better idea to leave it in there for the future in case we end up wanting to use it. I'll resolve this now.